webapp/src/components/sidebar_right/github_items.jsx

```diff
@@ -35,29 +37,140 @@ function GithubItems(props) {
                 </a>);
         }
 
+        let milestone = '';
+        if (item.milestone) {
+            milestone = (<span><i className='fas fa-bullseye'/>{item.milestone.title}</span>);
+        }
+
+        let reviewers = '';
+        if (item.reviewers && item.reviewers > 0) {
+            reviewers = (<span>{item.reviewers} pending reviews.</span>);
+        }
+
+        let reviews = '';
+        if (item.reviews) {
+            const reviewerUsers = [];
+            const filteredReviews = item.reviews.filter((v) => {
+                if (!reviewerUsers.includes(v.user.login)) {
+                    reviewerUsers.push(v.user.login);
+                    return true;
+                }
+                return false;
+            });
+
+            const approved = filteredReviews.reduce((accum, cur) => {
+                if (cur.state === 'APPROVED') {
+                    return accum + 1;
+                }
+                return accum;
+            }, 0);
+            const changesRequested = filteredReviews.reduce((accum, cur) => {
+                if (cur.state === 'CHANGES_REQUESTED') {
+                    return accum + 1;
+                }
+                return accum;
+            }, 0);
+
+            if (changesRequested > 0) {
+                reviews = (<span>Changes requested.</span>);
+            } else if (approved > 0) {
+                if (approved === 1) {
+                    reviews = (<span>{approved} approved review.</span>);
+                } else {
+                    reviews = (<span>{approved} approved reviews.</span>);
+                }
+            }
+        }
+
+        let status = '';
+
+        // Status images pasted directly from GitHub. Change to our own version when styles are decided.
+        if (item.status) {
+            switch (item.status) {
+            case 'success':
+                status = (
+                    <svg
```

Comment:
@larkox Can we move these svg's to a separate file with named exports?

@marianunez Does this make sense or do you think they should each have their own file? I think it clutters this file having them laid out like they are currently.

Reply:
+1 on their own file.